.env

```diff
@@ -1 +0,0 @@
-NEXT_PUBLIC_AGGREGATIONS_SPLIT=1
```

Comment:
let's delete this file and ignore it in .gitignore too. we have this https://github.com/liquidinvestigations/hoover-ui/blob/02d29a1baec4c92244e1613a5c71b9d3ce69350e/.gitignore#L9 which I guess is not used?